core/models2/Account.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
```

Comment:
@noahsthompson Brandon setup a new Django app - let's migrate these to there to keep the Creators app organized (create a 'models' directory in that app - 'models2' here isn't very descriptive)